src/styles/blocks/card.scss

```diff
@@ -0,0 +1,72 @@
+.card {
+  box-sizing: border-box;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  font-size: 10px;
+  color: $base-color;
+  line-height: 18px;
+  &__picture-img {
+    height: 134px;
+    display: flex;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $base-color;
+    margin: 0 0 4px;
+  }
+
+  &__code {
+    line-height: 14px;
+    color: $light-color;
+    margin-bottom: 16px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 22px;
+  }
+
+  &__price-wrap {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__price-lable {
+    font-size: 12px;
+    color: $light-color;
```

Comment:
The variable `$light-color` is used here again. Ensure that it is defined in a separate SCSS variables file and imported correctly.